src/rgw/rgw_op.cc

```diff
@@ -2597,9 +2597,11 @@ void RGWStatAccount::execute()
   string marker;
   rgw::sal::RGWBucketList buckets;
   uint64_t max_buckets = s->cct->_conf->rgw_list_buckets_max_chunk;
+  const string *lastmarker;
 
   do {
 
+    lastmarker = 0;
```

Comment:
crufty, musty K&R-ism bleeding into modern C++ code ; s/0/nullptr/

Reply:
In this case; compiler doesn't complain, and nullptr is a musty java-ism sneaking into modern c++ making things harder to read.

Reply:
Java uses _null_, which it got from C (after removing the all caps macro convention).

Reply:
it isn't -modern- c++, but tbf this was the preferred idiom in ISO c++ (1998), and prior.

Reply:
please use nullptr to be consistent with the project's [coding style](https://google.github.io/styleguide/cppguide.html#0_and_nullptr/NULL)